babel/messages/pofile.py

```diff
@@ -80,6 +80,47 @@ def denormalize(string: str) -> str:
         return unescape(string)
 
 
+def _extract_locations(line: str) -> list[str]:
+    """Extract locations from reference strings.
+
+    Locations are extracted while properly handling First Strong
+    Isolate (U+2068) and Pop Directional Isolate (U+2069), used by
+    gettext to enclose filenames with spaces and tabs in their names.
+    """
+    locations = []
+    location = ""
+    in_filename = False
+    for c in line:
+        if c == "\u2068":
+            if in_filename:
+                raise ValueError("reference contains more First Strong Isolate characters, "
+                                 "than Pop Directional Isolate characters")
```

Comment:
This might be a bit clearer?
```suggestion
                raise ValueError("File location contains more First Strong Isolate characters, "
                                 "than Pop Directional Isolate characters")
```

Reply:
I tried to stick to the wording gettext uses, that's why it's called reference there. While I agree that it might not be perfectly clear, I wonder if file location is really clearer.

Reply:
The CLI uses a combination of `location`, `location comments` and `file location`, but I don't think `reference` is ever used to refer to locations. Maybe `location comment` is better?

Reply:
I changed it to "location comment" now.